addon/styles/addon.scss

```diff
@@ -1 +1,79 @@
 @import 'node_modules/ember-frost-theme/scss/frost-theme';
+
+
+.frost-object-details {
+  display: flex;
+  flex-direction: column;
+  height: 100%;
+}
+
+.frost-object-details > .body{
+  flex: 1;
+  background-color: $frost-color-lgrey-3;
+  display: flex;
+  flex-direction: column;
+}
+
+.frost-object-details > .body > .top-tabs {
+  display: flex;
+  flex-direction: row;
+  height: 20px;
+  margin: 10px 0 0 10px;
+
+  a {
+    margin-right: 5px;
+  }
+  a:nth-child(n + 2) {
+    border-left: 1px solid black;
+    padding-left: 5px;
+  }
+  a.active {
+    color: $frost-color-blue-1;
+  }
+
+}
+
+.frost-object-details > .body > .body {
```

Comment:
That's a bit of a nasty selector chain - why do we have two classes called "body"?